src/clj/puppetlabs/services/request_handler/request_handler_service.clj

```diff
@@ -2,19 +2,28 @@
   (:require [puppetlabs.trapperkeeper.core :as tk]
             [puppetlabs.services.protocols.request-handler :as handler]
             [puppetlabs.services.request-handler.request-handler-core :as core]
+            [puppetlabs.trapperkeeper.services :as tk-services]
             [puppetlabs.services.jruby.jruby-puppet-service :as jruby]
-            [puppetlabs.trapperkeeper.services :as tk-services]))
+            [clojure.tools.logging :as log]))
 
 (defn- handle-request
```

Comment:
This whole function should be removed now, right, since nothing should be calling it?